core/src/test/java/org/elasticsearch/action/admin/indices/create/CreateIndexIT.java

```diff
@@ -367,9 +367,8 @@ public void testCreateShrinkIndexToN() {
     }
 
     public void testCreateShrinkIndex() {
-        internalCluster().ensureAtLeastNumDataNodes(4);
-        prepareCreate("source").setSettings(Settings.builder().put(indexSettings())
-            .put("number_of_replicas", 1).put("number_of_shards", 2)).get();
+        internalCluster().ensureAtLeastNumDataNodes(2);
```

Comment:
I agree but isn't it taken care of by `InternalTestCluster#DEFAULT_MAX_NUM_DATA_NODES`?